lib/index.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const _get = require('lodash.get');
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const {promisify} = require('util');
+const exec = promisify(require('child_process').exec);
+const path = require('path');
+const {util: {BedrockError}} = bedrock;
+
+const bedrockPackages = {};
+
+bedrock.events.on('bedrock.init', async () => {
+  let stdout;
+  try {
+    ({stdout} = await exec('npm ls --parseable --silent'));
```

Comment:
I'm concerned about the sure-to-be-annoying startup cost of doing this.  First run:
```
.../random-bedrock-project$ time npm ls --parseable --silent
real    0m8.183s
```
Caching or whatever will make it faster later:
```
real    0m2.250s
```
I think my setup is a bit quirky, so those may be high, but it's still something that will slow things down.

Also there is a `--json` output option.

Reply:
They also had an API, `npm.commands.ls(...)`, but maybe that was deprecated?  Docs not at their old location.

Reply:
I spot checked a few top level modules, and I'm seeing max value of ~5 sec with the mean being closer to 3 sec.  

Here is sample --json output which unfortunately does not include arbitrary properties from package.json files: https://gist.githubusercontent.com/mattcollier/203191d7407dd27dd6526eb6caf2f39e/raw/e89316c8c8059a5b6a8c7526efb27e44ca7fd185/some.json

After I found the following post indicating that the NPM nodejs API is a use at your own risk sort of thing, I abandoned the search: https://stackoverflow.com/a/15957574

Reply:
Yikes -- waiting seconds for startup could easily get annoying fast, particularly when developing. Sounds like we should target another method to get the information we need or have a good mitigation strategy.